packages/docs/docs/docs/migrate_from_1.x_to_2.x.md

```diff
@@ -0,0 +1,117 @@
+---
+id: migrate_from_1.x_to_2.x
+title: Migrating to 2.x
+---
+
+In this section, you'll find everything you need to know if you're coming from ZeppelinOS 1.x and want to start using 2.x.
+
+## Terminology and fundamentals
+
+2.x's most fundamental change is that it expands on zOS's linking features. In 1.x, you were able to link your projects to what used to be called an "on-chain standard library". This "stdlib" was basically a pre-deployed version of OpenZeppelin, which gave your projects instant access to hundreds of standard implementations without having to compile or deploy any code whatsoever. This proved to be a very powerful, innovative concept, but we realized that it could be taken much further... In 1.x, you could connect to different versions of this "standard library" but there was just this ONE "standard library" that your application could link to. 2.x takes this to a whole new level; you can now connect your project to multiple "standard libraries", but that's not all, this makes the entire concept of what used to be a "standard library" evolve to the concept of an "EVM package".
+
+What is an EVM package? It is basically a piece of reusable code that has been deployed to the blockchain in the form of EVM bytecode. Anyone can create an EVM package, and your zOS projects will be able to connect to a vast ecosystem of pre-compiled, pre-deployed reusable code in the form of EVM packages.
+
+Similarly, as we realized what an EVM package was, we realized what it's not. In 1.x, your zOS project's contracts were always managed by two contracts: App and Package. This made your contracts upgradeable and gave them the ability to link to EVM packages (see [Contract architecture](https://docs.zeppelinos.org/docs/architecture.html) for more info). We realized that these two contracts made sense when it came to EVM packages, but didn't when it came to parts of your project that weren't meant to be reusable code, i.e. code that just needed to be upgradeable but not reusable. And so, in 2.x, upgradeability, and the ability to link to EVM packages were completely separated.
+
+In 2.x by default, your zOS project will not use any of its App or Package on-chain architecture unless you explicitly state that your code is meant to be reusable as an EVM package. When you init a zOS project, there is no architecture other than your project's code. ZeppelinOS simply uses the CLI to manage proxies for you. Now, if you intend your project to be a reusable EVM package, you can `publish` your code, and the CLI will seamlessly deploy the necessary App and Package architecture which will allow your code to exist as a discoverable deployed EVM package.
+
+## New commands
+
+2.x introduces a few new commands to its CLI which are described below:
```

Comment:
The commands reference will be in https://docs.zeppelinos.org/docs/cli_main.html
so, I would remove the details, and just link to that page.